src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleSurfaceViewRenderer.cs

```diff
@@ -0,0 +1,128 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using ElmSharp;
+using ElmSharp.Wearable;
+using Tizen.Wearable.CircularUI.Forms;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+using XForms = Xamarin.Forms.Forms;
+using ELayout = ElmSharp.Layout;
+
+[assembly: ExportRenderer(typeof(CircleSurfaceView), typeof(Tizen.Wearable.CircularUI.Forms.Renderer.CircleSurfaceViewRenderer))]
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class CircleSurfaceViewRenderer : ViewRenderer<CircleSurfaceView, Box>
+    {
+
+        Dictionary<ICircleSurfaceItem, ICircleWidget> _circleSurfaceItems;
+        ELayout _surfaceLayout;
+        CircleSurface _circleSurface;
+
+        protected override void OnElementChanged(ElementChangedEventArgs<CircleSurfaceView> e)
+        {
+            if (Control == null)
+            {
+                var box = new ObservableBox(XForms.NativeParent);
```

Comment:
Why did you use ObserableBox instead of simple Native.Box or Native.Canvas?

Reply:
good point.